### DISCUSSION TITLE
Compatible camera

### DISCUSSION THREAD

**Discussion by penzmann** [photobooth-app/photobooth-app#270] [Howtos and camera specific topics]
Is a Sony Alpha 6000 compatible?

**Comment by Striker62** [comment]
you can find a list of supported cameras here 
http://gphoto.org/proj/libgphoto2/support.php
The Sony Alpha  6000 is in the list but no features listed .
I would try

**Reply by penzmann** [reply]
Thank you for the answer.

I know the list. What does no features mean?

**Reply by Striker62** [reply]
Your Camera has 2 entries 
(Control ) and (MTP) 
I've not seen the (Control) here some features listed


<html><body>

Sony Alpha-A5100 (MTP) |   |  -- | -- | --

Sony Alpha-A6000 (Control) | Image Capture, Trigger Capture, Configuration |  

Sony Alpha-A6000 (MTP) |   |  

</body>
</html>


I have a nikon D3100
<html><body>

Nikon DSC D3100 (PTP mode) | Image Capture, Trigger Capture, Configuration
-- | --



</body>
</html>

Works perfect , I'm not shure what the difference between Control and PTP is.

**Reply by Striker62** [reply]
you also can look hrer http://gphoto.org/doc/remote/

if you have installed ghopto2 and connect your camera, you can use 

the command --list-config. That should show the features the camera provide

**Reply by mgineer85** [reply]
You need to try or search gphoto and your camera name to get an indicator. We rely on that and have no additional information about your specific camera model unfortunately.

**Reply by penzmann** [reply]
Many thanks for the answer!

**Reply by penzmann** [reply]
Hello!
I have installed everything so far. However, there is no connection between the Raspberry Pi and my camera (Sony Alpha 6000).   

What settings do I need to make on the camera?

Thank you very much!

**Comment by Striker62** [comment]
Hi,

what did you get by command lsusb ?
I get for my NIKON 

Bus 001 Device 004: ID 04b0:0427 Nikon Corp. NIKON DSC D3100

**Reply by penzmann** [reply]
It is listed for me:

Bus 003 Device 002: ID 054c:08B7 Sony Corp. ILCE-6000 (alias Alpha 6000) in MTP mode

And other USB-Devices..Keyboard..

Is MTP mode ok?

**Reply by Striker62** [reply]
As far as I know is MPT windows compatible.
My Nikon support PTP what works for me .

anyway you have a connection between the sony and the pi.
Did you setup the backend part in photobooth to gphoto2 ?

For me I had to use a Webcam for the liveview because my nikon does not support liveview.

**Reply by Striker62** [reply]
I had to remove the SD card from my Camera to get it work.

**Reply by penzmann** [reply]
PTP mode is not available on my camera...

**Reply by penzmann** [reply]
Setup the backend: How do I do that?

**Reply by penzmann** [reply]
Would you be so kind as to send me a step-by-step guide? Either by video or the individual commands.

Unfortunately, I'm still a beginner.

I would be very grateful, because I have a wedding in two weeks where I would like to set up the photo box.

**Reply by Striker62** [reply]
Yes that is what the gphoto page say but control is listed.

Sony Alpha-A6000 (Control) | Image Capture, Trigger Capture, Configuration |  

I have no idea if you can switch the mode but I would try

**Reply by Striker62** [reply]
In the photobooth admin page you hace to setup the Camera ( Backend ). Michael has descriped this in the manual

**Reply by Striker62** [reply]
you can also find a hint what can be also a reason why the camera is not responding 

https://photobooth-app.org/support/faq/
Gphoto2 camera found but no access

**Reply by Striker62** [reply]
<<<<< would be very grateful, because I have a wedding in two weeks where I would like to set up the photo box.

if this is the case i recoment to use a good webcam. this is easy to setup

**Reply by Striker62** [reply]
i found for sony camera a document that descripe to setup the usb connetion but this is not specific for your camera
please have a look in the manual of your camera how to set the usb connection to pc controlled.
this can probaly help. but you will anyhow need a usb camera because your camera is not supporting liveview as mine.

**Reply by mgineer85** [reply]
Thank you striker for helping 👍
There is nothing we can do, if gphoto work we can use it, if it doesn’t work we have no option. So it is about googling gphoto and your camera model to check if there is more information

**Reply by penzmann** [reply]
I have ordered a camera which I will receive today.

I'm going to try removing the memory card today.

Many thanks for your support!

**Reply by PixelAlarm** [reply]
I used Sony 6000 before.
If help is needed, I could borrow from a friend and help debug.
But 6000 hasn‘t got live view, just to keep in mind.
